### DISCUSSION TITLE
download/free/dce5-installer-v0.14.0

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3551] [Announcements]
# download/free/dce5-installer-v0.14.0

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/download/free/dce5-installer-v0.14.0.html

**Comment by samzong** [comment]
有些情况下，用户下载是需要到服务器下载的，建议参考解压缩命令，给出 curl 或者 wget 命令:

```bash
curl -C - -O https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/dce5/offline-community-v0.14.0-amd64.tar
```
